humanize-duration.js

```diff
@@ -507,7 +507,7 @@
     // Has the nice sideffect of turning Number objects into primitives.
     ms = Math.abs(ms)
 
-    var dictionary = options.languages[options.language] || languages[options.language]
+    var dictionary = options.languages[options.language] || languages[options.language] || languages[options.fallback]
```

Comment:
IMO, this should pull from both `options.languages` and `languages`. Maybe something like this?

```suggestion
    var dictionary
    if (options.languages.hasOwnProperty(options.language)) {
      dictionary = options.languages[options.language]
    } else if (languages.hasOwnProperty(options.language)) {
      dictionary = languages[options.language]
    } else if (options.languages.hasOwnProperty(options.fallback)) {
      dictionary = options.languages[options.fallback]
    } else if (languages.hasOwnProperty(options.fallback)) {
      dictionary = languages[options.fallback]
    }
```

This is a bit verbose, but also fixes a bug (passing a `language` like `hasOwnProperty` and watching everything crash).

Reply:
Are you saying to replace the option `language` with `languages`? If so, that would be a breaking change. I think we could get away with keeping it as language that way it'll just be a minor bump. Unless you feel strongly about it, of course?